src/domainLogic/index.ts

```diff
@@ -10,3 +10,14 @@ export function calcShare(
 ): PercentAmount {
   return new PercentAmount(bnToBn(currentValue)).div(bnToBn(fullValue)).mul(100);
 }
+
+export function calcLoanAPY(myInterestShare: PercentAmount, loanAPR: PercentAmount): PercentAmount {
+  return new PercentAmount(myInterestShare).mul(loanAPR).div(100);
+}
+
+export function calcLoanProfit(
+  myAPY: PercentAmount,
```

Comment:
maybe just use `apy` or `stakeCost`, or add `user` before each of the params? `my` doesn't mean anything to me